.travis.yml

```diff
@@ -130,6 +129,18 @@ before_install:
     if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       brew update
       BUILD_FLAGS="$BUILD_FLAGS -DSHARED_LIBS_ABS_PATH=ON"
+      # If ITK is required, use conda
+      if [[ $EXTRA_BUILD_FLAGS == *"-DUSE_ITK=ON"* ]] && [[ $EXTRA_BUILD_FLAGS == *"-DUSE_SYSTEM_ITK=ON"* ]]; then
+        echo "Installing ITK via miniconda"
+        wget https://repo.continuum.io/miniconda/Miniconda$PYMVER-latest-MacOSX-x86_64.sh -O miniconda.sh;
+        bash miniconda.sh -b -p $HOME/miniconda
+        export PATH="$HOME/miniconda/bin:$PATH"
+        hash -r
+        conda config --set always_yes yes --set changeps1 no
+        conda update -q conda
+        conda install -c conda-forge libitk
+        EXTRA_BUILD_FLAGS="$EXTRA_BUILD_FLAGS -DITK_DIR=$HOME/miniconda/pkgs/itk-4.12.0-0/lib/cmake/ITK-4.12"
```

Comment:
this is incorrect as conda is now at libitk-4.13.0. It seems in any case that the line is not needed as CMake is picking it up, as I see in the Travis log file a mention of `/Users/travis/miniconda/include/ITK-4.13/itkImage.h`